pallets/avn-proxy/src/lib.rs

```diff
@@ -0,0 +1,226 @@
+//! # Avn proxy pallet
+// Copyright 2020 Aventus Network Services (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2022 Aventus Network Services (UK) Ltd.
```
Probably worth updating the copyright here